### DISCUSSION TITLE
Can't figure out what's wrong in using sourceMapFilename in the output object

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#13038] [Q&A]
I am experiencing a frustrating bug finding it hard to figure out the solution. I did refer to some other git discussions which have the error message as :

**ERROR in Conflict: Multiple assets emit different content to the same filename main.map**

But what's more frustrating is I can't find a solution for this for webpack 5. If I remove the sourceMapFilename from my webpack config it complains about missing files in the broswer when loading the page. 

Please help!

**Comment by alexander-akait** [comment]
Please stop spamming, again - **Please create readable reproducible test repo**

**Comment by sokra** [comment]
Use `[file]` in that template, otherwise is and css generate the same source map name
